inst/BS5/assets/pkgdown.scss

```diff
@@ -103,6 +113,9 @@ aside {
   .list-unstyled li { margin-bottom: 0.5rem; }
   .dev-status .list-unstyled li { margin-bottom: 0.1rem; }
 }
+[data-bs-theme="dark"] aside .roles {
```

Comment:
I think it makes the most sense to define the dark styles next to the corresponding light styles rather than one big block at the end, because basically where ever we define a light colour, we're also going to need to define a dark one.